docs/src/pages/[platform]/connected-components/authenticator/customization/customization.override-function-calls.react-native.mdx

```diff
@@ -9,55 +9,8 @@ To override a call you must create a new `services` object with an `async` `hand
 The service object must then be passed into the `authenticator` component as a `services` prop. For example, let's imagine you'd like to lowercase the `username` and the `email` attributes during `signUp`.
 This would be overriden like so:
 
-<Alert role="none" variation="warning" heading="Sign Up Auto Sign In">
-  When overriding `signUp` you must include the `autoSignIn` key and set `enabled` to true, as show in the example below.
-</Alert>
+<ReactNativeOverrideExample />
 
-```jsx expoSnack file=../../../../../../../examples/react-native/src/features/Authenticator/OverrideServices/Example.tsx
+Each `handle*` function will return the neccessary values you'll need to make the call to the `Auth` function call. Here is a table of each override function name, and the values returned from `formData`.
```

Comment:
That's what it already said, I just edited `Auth.*` -> `Auth`. I can update the wording but if we wanted to avoid scope creep should we save it for a future pr?